scipy/special/special/legendre.h

```diff
@@ -2,6 +2,8 @@
 
 #include "config.h"
 
+extern "C" double cephes_poch(double x, double m);
+
```

Comment:
You know `poch` is already translated right? https://github.com/scipy/scipy/blob/main/scipy/special/special/cephes/poch.h

Reply:
I didn't know that! Great, done.